setup.py

```diff
@@ -1,69 +1,6 @@
 #! /usr/bin/env python
```

Comment:
My sense is that everything will work as expected if this file is deleted.

Reply:
I kept it for legacy purposes / compat with existing calls like `python setup.py check` and `python setup.py develop` by following the `Important 🔥 ` note at the top here

https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html